kolibri/plugins/learn/assets/src/vue/search-widget/index.vue

```diff
@@ -237,6 +237,7 @@
     background-color: $core-bg-canvas
     position: fixed
```

Comment:
This fixed the issue on chrome, haven't tested on other browser s yet .
Avoid position:fixed ? For what reason ?